### ISSUE TITLE
No Aria-label in create pages edit add feature-image button

### ISSUE THREAD

**Issue by AfLosada** [AfLosada/Ghost_MISO#28]
**Describe the bug**
No Aria-label in create pages edit add feature-image buttons. This should be enabled to facilitate navigation for users with screen-readers.

**To Reproduce**
1. Create Page
2. Click on button for feature-image with inspector
4. See that it has no aria-label

**Expected behavior**
Every button should have an aria-label that should help screen-readers detect it

**Screenshots**
[VIDEO](https://youtu.be/orHsyn88_ys)

**Environment (please complete the following information):**
 - Device: DESKTOP
 - OS: Windows 11
 - Browser: Chrome
 - Version 5.68.0
